frontend/src/bundles/common/components/badge/badge.tsx

```diff
@@ -14,6 +14,8 @@ type Properties = {
     color?: ValueOf<typeof BadgeColors>;
     small?: boolean | undefined;
     icon?: ReactElement;
+    iconClass?: string;
+    HRbadge?: boolean;
```

Comment:
Name boolean variable like is..., has... etc

Reply:
done